RuntimeLibrary/plsan_handler.h

```diff
@@ -0,0 +1,20 @@
+#ifndef PLSAN_HANDLER_H
+#define PLSAN_HANDLER_H
+
+#include "tuple"
+
+namespace __plsan {
+
+enum AddrType { NonDynAlloc, DynAlloc };
+enum ExceptionType { None, RefCountZero, DynAllocMemNonFree };
```

Comment:
RefCountZero and DynAllocMemNonFree are the same thing. 
Except that, anything else LGTM

Reply:
@purplepig4657 
Oh, but Please ensure that it remains possible to build
 We don't want to break what worked before

Reply:
@hygoni 
Please check changed CMakeLists.txt